daemon/src/monitor.rs

```diff
@@ -722,7 +723,7 @@ impl MonitorParams {
         MonitorParams {
             lock: (dlc.lock.0.txid(), dlc.lock.1),
             commit: (dlc.commit.0.txid(), dlc.commit.2),
-            cets: map_cets(dlc.cets),
+            cets: map_cets(dlc.cets, &dlc.maker_address),
```

Comment:
Is it correct to always use the `maker` address here? 
I suppose it should not matter, but am not sure.

Reply:
I think the thinking is that the maker's script will always be present whereas the taker's address might not be in the case of a liquidation. This deserves a comment though :)